### PR TITLE
Fix Swagger JSON autogeneration issues.

### PR DESCRIPTION
**Pull request by zeripath**
When autogenerating a swager client from the current swagger.v1.json, at least in Java, there is an issue whereby the use of schema references for the forbidden and empty responses leads to a broken client being created due to swagger believing that these represent an unspecified object. This pull requests removes these unnecessary schema references for the API references where a description for the empty/forbidden responses is provided.

### CODE REVIEW

**Comment by sapk** [comment]
I remember the opposite problem with go version of swagger but since CI doesn't seems to complain. LGTM

**Review by sapk — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
Most of the swagger api descriptions incorrectly use the integer type when the underlying go code actually uses int64. I've added further changes to account for this.

**Comment by zeripath** [comment]
I've also noted that the swagger description of GET /repos/{owner}/{repo}/pulls was also inadequate

File: routers/api/v1/admin/org.go [hunk 1]
```diff
@@ -28,6 +28,10 @@ func CreateOrg(ctx *context.APIContext, form api.CreateOrgOption) {
 	//   description: username of the user that will own the created organization
 	//   type: string
 	//   required: true
+	// - name: organisation
```
Comment (lunny) [inline routers/api/v1/admin/org.go]: organisation or organization ?

**Review by lunny — CHANGES_REQUESTED** [review]

File: routers/api/v1/admin/org.go (hunk 1 above)
Comment (zeripath) [reply routers/api/v1/admin/org.go]: Sorry, I've fixed that. I presume we're standardising on US spelling rather than UK or International spelling...

**Comment by lafriks** [comment]
@lunny need your approval

**Review by lunny — APPROVED** [review]

**Pull request merged**